doc/CHANGELOG

```diff
@@ -5,6 +5,7 @@ versions (where they are missing from early versions, assume that they
 should be attributed to Megan Potter).
 
 KallistiOS version 2.1.0 -----------------------------------------------
+- *** Add support for compiling with LTO [Paul Cercueil = PC]
```

Comment:
New entries in the CHANGELOG go at the bottom of the list for that version, not the top (otherwise we'd be constantly moving the = entries around.

Reply:
Updated.